topic_folders/policies/immediate-response-checklist.md

```diff
@@ -0,0 +1,74 @@
+## Carpentries Staff Immediate Response Checklist
+
+When a Code of Conduct (CoC) violation is found, one course of action is that activities in Carpentries channels (in-person and virtual) are monitored. In other cases, the community member is asked to take a termed suspension. When this is the resolution of the CoC committee, the following boundaries are enforced. The checklist below was created for incidents that require an immediate response, i.e. the community member is removed from all Carpentries engagement.
+
+The intended outcomes for such a suspension are immediate, ongoing, and demonstrable changes in any areas of engagement for which the CoC deemed undesirable in the incident review. 
+
+### Online Communication and Communities
+Revoke access from the following communication channels and communities: 
+- [ ] Email lists on TopicBox 
+- [ ] Email lists on Google Groups
+  - [ ] Library Carpentry Governance Group
+  - [ ] Library Carpentry Maintainer Onboarding
+  - [ ] Code of Conduct Committee
+  - [ ] Carpentries Mentoring Groups 
+- [ ] Twitter - (@datacarpentry, @swcarpentry, @libcarpentry, @thecarpentries)
+- [ ] Blogs - (Data Carpentry, Software Carpentry, Library Carpentry, The Carpentries)
+- [ ] GitHub (carpentries, datacarpentry, swcarpentry, librarycarpentry, data-lessons, carpentrieslab)
+- [ ] Gitter - Library Carpentry Lobby
+- [ ] [Slack](https://swcarpentry.slack.com/messages)  
+- [ ] Facebook 
+- [ ] LinkedIn 
+
+Additionally, the community member will not be allowed to attend community discussions.
+
+### Teaching Workshops
+The suspended member will, for the duration of their suspension:
+- [ ] Be asked not to sign up for workshops on the Instructor Spreadsheet.
+- [ ] Not be able to participate as a helper at any workshops.
+- [ ] Be removed from Eventbrite login access.
+- [ ] Not be allowed to plan self-organized workshops.
+- [ ] Not be allowed to request or host centrally coordinated workshops.
+- [ ] Not be listed as an instructor on the “Instructors” webpage.
+- [ ] Be removed from the instructors email list.
+
+### Organizing Workshops 
+If the suspended member is a Workshop Administrator or serves in another role whereby they organize workshops for The Carpentries, for the duration of their suspension they will not have access to:
+- [ ] LastPass
+- [ ] AMY
+- [ ] Regional Email
+- [ ] Asana
+- [ ] HelpScout
+- [ ] Zoom
+- [ ] Workshop Administrator’s Slack Channel
+- [ ] Workshop Administrator’s Email List
+
+### Instructor Training
+The suspended member will, for the duration of their suspension, be removed from the “Our Trainers” web page. Additionally, the suspended member will not be allowed to: 
```

Comment:
I know I might contradict myself a bit here, but I think this one is ok. We should hold our trainers to a higher standard, and thus probably have more transparency at that level.